hivemq-edge/src/frontend/src/extensions/datahub/components/forms/ReactFlowSchemaForm.tsx

```diff
@@ -1,4 +1,4 @@
-import { FC, useMemo } from 'react'
+import { FC } from 'react'
```

Comment:
note(non-blocking): you can also import types like this from a module with that there is no trace of it in the runtime.

> `import type` only imports declarations to be used for type annotations and declarations. It always gets fully erased, so there’s no remnant of it at runtime. Similarly, export type only provides an export that can be used for type contexts, and is also erased from TypeScript’s output.

https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-8.html#type-only-imports-and-export

```suggestion
import type { FC } from 'react'
```

Reply:
Can also be done via TypeScript-ESLint automatically.
https://typescript-eslint.io/rules/consistent-type-imports/

Reply:
Yes, good reminder. No more CR on this ticket: there are 11150 of them in 600 files :-)